### PR TITLE
Creates attendance scaffold

### PR DESCRIPTION
**Pull request by JuanequeX**
# Description
 - Creates the scaffolding for attendances in the project
 - Set the time zone for gets a correct time in the check in and check out
 - I Implemented models and request tests for carry out a analyzed the values
 
 # What?
 I've added the attendance scaffolding to carry out of reports of attendances per day, average time by months attendances and absence by month

 # Why?
Because we need a structure inside the project to administrate the attendance data

 # Testing
Can you testing in the file `spec/models/attendance_spec.rb` and `spec/requests/attendances_spec.rb`

## Desktop
desktop
_____________________________________________________________________
attendance-desktop

## Mobile
mobile
______________________________________________________________________
attendance-mobile

### CODE REVIEW

File: app/models/employee.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
 class Employee < ApplicationRecord
-  acts_as_paranoid 
+  acts_as_paranoid
   belongs_to :user
   validates :name, :email, :private_number, presence: true
   validates :email, :private_number, uniqueness: true
   validates :private_number, numericality: { only_integer: true }
-  
+
```
Comment (YilKanda) [inline app/models/employee.rb:7]: Here's an extra line.

Comment (YilKanda) [inline app/models/employee.rb:7]: Here's an extra line.

Comment (JuanequeX) [reply app/models/employee.rb:7]: i've deleted this line, thanks

**Review by adrianMRJ — APPROVED** [review]
LGTM!

**Pull request merged**